### PR TITLE
Change toolset from VS2015 v140 to VS2017 toolset v141

### PR DESCRIPTION
**Pull request by karthiknadig**

### CODE REVIEW

File: src/Microsoft.R.Host.vcxproj [hunk 1]
```diff
@@ -33,29 +33,29 @@
     <ConfigurationType>Application</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
     <PlatformToolset Condition="'$(VisualStudioVersion)'=='14.0'">v140</PlatformToolset>
```
Comment (MikhailArkhipov) [inline src/Microsoft.R.Host.vcxproj]: You can safely remove 14.0, we no longer support it in master

**Comment by MikhailArkhipov** [comment]
So boost finally produced 141 build?

**Comment by MikhailArkhipov** [comment]
👍

**Comment by karthiknadig** [comment]
This is not needed.

**Pull request closed without merging**